build-support/controller/README.md

```diff
@@ -0,0 +1,9 @@
+# Consul-K8s hack GuideLines
+
+This document describes how you can use the scripts from [`hack`](.) directory 
+and gives a brief introduction and explanation of these scripts. 
+
+## Overview
+
+The [`hack`](.) directory contains many scripts that ensure continuous development of consul-k8s, 
+enhance the robustness of the code, improve development efficiency, etc. 
```

Comment:
This isn't the hack directory anymore.

I think this is cool to just say (for now), "`boilerplace.go.txt` is needed for the operator-sdk. It allows for specifying boilerplate to be added to the top of each generate file. It is empty because we do not need any boilerplate but the file must exist for the generation to work"

Reply:
lol. this is what i get for copypasting from kubernetes and not taking a second to read what i just copied over.